06-improve-model-performance/improvement-overview.ipynb

```diff
@@ -0,0 +1,913 @@
+{
```

Comment:
<p>There's a much easier way to do this...</p><p> </p>

<pre class="ql-syntax" spellcheck="false">eval_results = client.agents.datasets.evaluate.retrieve(dataset_name=eval_status.dataset_name, agent_id=agent_id)
eval_results.write_to_file('eval_results.json')

with open('english_eval_results.json', 'r') as f:
    eval_objects = [json.loads(line) for line in f]

df = pd.DataFrame(eval_objects)
df
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ContextualAI/examples/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>good thought, but the scores are buried in nested json and this doesn't get them out</p>